src/components/user/user.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { TypesOfTodoList } from '../../types/type';
+import './user.scss';
+
+export const User = ({ task }) => (
+  <p className="user-name">
+    <i>{task.user}</i>
```

Comment:
Давай конкретні назви для пропси
```suggestion
export const User = ({ task }) => (
  <p className="user-name">
    <i>{task.user}</i>
```